Detectors/TPC/reconstruction/macro/runCATracking.C

```diff
@@ -50,33 +44,44 @@ void runCATracking(TString filename, TString outputFile, TString options, Int_t
   TChain c("cbmsim");
   c.AddFile(filename);
 
-  TClonesArray *clusters=0x0;
-  c.SetBranchAddress("TPCClusterHW", &clusters);
-
   // ===| output tree |=========================================================
   TFile fout(outputFile, "recreate");
   TTree tout("events","events");
   tout.Branch("Tracks", &tracks);
 
-  // ===| event ranges |========================================================
-  const Int_t nentries = c.GetEntries();
-  const Int_t start = startEvent>nentries?0:startEvent;
-  const Int_t max   = nmaxEvent>0 ? TMath::Min(nmaxEvent, nentries-startEvent) : nentries;
-
-  // ===| loop over events |====================================================
-  for (Int_t iEvent=0; iEvent<max; ++iEvent)   {
-    c.GetEntry(start+iEvent);
-
-    printf("Processing event %d with %d clusters\n", iEvent, clusters->GetEntries());
-    if (!clusters->GetEntries()) continue;
-
+  if (mergeChain) {
     tracks.clear();
-    if (tracker.runTracking(clusters, &tracks) == 0)     {
+    printf("Processing full TChain of clusters at once\n");
+    if (tracker.runTracking(&c, &tracks) == 0)     {
       printf("\tFound %d tracks\n", (int) tracks.size());
     } else {
       printf("\tError during tracking\n");
     }
     tout.Fill();
+  } else {
+    TClonesArray *clusters=0x0;
```

Comment:
please use `nullptr` instead of `0x0`. The codechecker should have complained about this .. sigh.